data/data/agent/systemd/units/iscsi-oci-configure-secondary-nic.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Configure secondary NIC on OCI when using iSCSI boot drive
+After=NetworkManager.service
+
+[Service]
+StandardOutput=syslog
+StandardError=syslog
+ExecStart=/usr/local/bin/iscsi-oci-configure-secondary-nic.sh
```

Comment:
Although the script checks the chassis_asset_tag to check if its run in OCI, could it be more readable if we add a condition to only start this service if platform is OCI

Reply:
I agree, but there doesn't seem to be a suitable condition type for determining that directly. So we'd end up having to run another service to create the thing that we could make this conditional on. It's doubtful _that_ would be more readable.